README.md

```diff
@@ -168,40 +159,12 @@ This theme is designed to be used as-is within actual **VC Storefront**. You can
      - `Launchpad`, `Finder`, `Homebrew` or `MacPorts` on *macOs*
      - Native package manager such as `apt` on *Linux*
 
-### Install the `vc-storefront`
-
-- Clone [https://github.com/VirtoCommerce/vc-storefront](https://github.com/VirtoCommerce/vc-storefront) in to a local folder
-- Open the **appsettings.json** file in a text editor
-- In the **Endpoint** section change **Url**, **UserName**, **Password** with correct path and credentials for Virto Commerce Platform:
-
-```
-  ...
-  "Endpoint": {
-    "Url": "https://localhost:5001",
-    "UserName": "admin",
-    "Password": "store",
-  ...
-```
-
-### Run `vc-storefront` application
-
-```bash
-# change the current directory
-cd C:\vc-storefront\VirtoCommerce.Storefront
-# build and run storefront application
-dotnet run
-# In future, if you don't need to rebuild you can use that
-dotnet run --no-build
-```
-
 ### Clone repository
 
 ```bash
-# Clone repo into the folder where storefront is installed
+# Clone repo 
 # `store-code` can be found in the platform running locally. More -> Shops -> Shop Name -> Code
```

Comment:
This line should be deleted - as it relates to the storefront

Reply:
fixed